src/index.ts

```diff
@@ -147,12 +151,11 @@ const processRenderResponse = (
 
         if (rollupOptions.insert) {
           /**
-           * Add `insertStyle` import for handling "inserting"
-           * *.css into *.html `head`.
-           * @see insertStyle.ts for additional information
+           * Include import using {@link INSERT_STYLE_ID} as source.
+           * It will be resolved to insert style function using `resolvedID` and `load` hooks
            */
-          imports = `import ${insertFnName} from '${__dirname}/insertStyle.js';\n`;
-          defaultExport = `${insertFnName}(${out});`;
+          imports = `import ${INSERT_STYLE_ID} from '${INSERT_STYLE_ID}';\n`;
```

Comment:
There is no need for relative path: function code is provided directly to rollup using `resolveID` and `load`, where the later returns the insertStyle function code.  
Rollup will output the result inside a `_virtual` folder (if using `preserveModules` option). 
I provided an example in the PR description.

Reply:
Correct, though rollup does not provide a runtime so how do runtime modules know how to resolve the _virtual module when using `preserveModules` feature (meaning the `insertStyle` function code is not included with all code in one file)?  Does rollup replace the import path with the "relative" one to the module?  If it doesn't then the module would be "not found" since the import engine doesn't know how to find the pathless one.

Reply:
> how do runtime modules know how to resolve the _virtual module when using preserveModules feature?

The import path of insertStyle is completely handled by rollup using relative imports in the produced output. 
You can see that in the screenshot I posted on the PR description...

Sorry but I'm having trouble understand what is the problem here.

Reply:
I hear though, then if that is the case we should have a proper "local" import path here (doesn't break the spec/conventions, and since rollup will handle the path any it doesn't hurt and helps to stop any confusion).

Reply:
Maybe if we can add a comment near there also letting folks know that rollup will rewrite the path with the user's build folder relative one - then it will be clearer what the code does.